src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -870,16 +891,14 @@ private boolean cancelScheduledTrip(
    * exist, and will be reused if a similar added/modified trip message is received with the same
    * route and stop sequence.
    *
-   * @param tripId      trip id without agency id
-   * @param serviceDate service date
    * @return true if a previously added trip was cancelled
    */
   private boolean cancelPreviouslyAddedTrip(
```

Comment:
The naming of this method is slightly misleading.

Reply:
I've limited what this method does so the naming is more correct now. I think this method was now otherwise doing duplicate stuff with the new clean-up method I've introduced to the GTFS RT logic.

Reply:
I now realize that https://github.com/opentripplanner/OpenTripPlanner/pull/5726/commits/1cb32d978a31ef0e244ad666d0922079650a3b2a commit title is unreadable but I don't think I'm going to fix it as we usually don't fix typos in commit titles :sweat_smile: